btb/benchmark/challenges/bohachevsky.py

```diff
@@ -53,3 +55,15 @@ def get_tunable_hyperparameters(self):
     def evaluate(self, x, y):
         z = 0.3 * np.cos(3 * np.pi * x)
         return -1 * (x**2 + 2 * y**2 - z - 0.4 * np.cos(4 * np.pi * y) + 0.7)
+
+    def __repr__(self):
+        args = inspect.getargspec(self.__init__)
```

Comment:
Maybe this could be implemented only once, in the base class?

Reply:
:+1:  I adapted `HyperParameters` aproach, but those challenges definitely can have the `__repr__` in their base class.